src/pages/AssetViewPage/AssetView.vue

```diff
@@ -45,33 +48,37 @@
           !isObjectDetailsOpen && !isAssetDetailsOpen, // neither panels open
       }"
       :showToggle="permitDrawerToggle"
-      :moreLikeThisItems="moreLikeThisItems"
       :objectId="assetStore.activeObjectId"
       :isOpen="permitDrawerToggle ? isObjectDetailsOpen : true"
       @toggle="isObjectDetailsOpen = !isObjectDetailsOpen"
     />
   </div>
 </template>
 <script setup lang="ts">
-import { ref, computed } from "vue";
+import { ref, defineAsyncComponent } from "vue";
 import { useAssetStore } from "@/stores/assetStore";
 import ObjectViewer from "@/components/ObjectViewer/ObjectViewer.vue";
 import ObjectDetailsDrawer from "@/components/ObjectDetailsDrawer/ObjectDetailsDrawer.vue";
 import AssetDetailsDrawer from "@/components/AssetDetailsDrawer/AssetDetailsDrawer.vue";
-import { useMoreLikeThis } from "@/helpers/useMoreLikeThis";
 import { useMediaQuery } from "@vueuse/core";
 
-const props = defineProps<{
+const MoreLikeThis = defineAsyncComponent(
```

Comment:
Why is this one an async component?

Reply:
Async component allows us to:
1. Skip loading code that's not needed.
2. Add a delay if we want.
3. Add an intersection observer and lazy load if we want.

I didn't know if the load performance would be satisfactory to you, so I made this as a async component in case I needed to tweak further. 

It's trivial to switch to a regular import if you have a preference. I don't really.

Reply:
Maybe I'm misunderstanding how Vue handles this - will it only load the async component if the <morelikethis> element is actually being rendered (IE, if the v-if resolves to true)? That'd be super cool. I was assuming this would cause the MoreLikeThis to be loaded on every AssetView regardless?

Slightly larger bundle size / longer initial load is overall preferable vs the latency and connection limiting on the server side. Can definitely re-examine this when we can move the servers to HTTP/2, which reduces that overhead.